lms/app.py

```diff
@@ -35,6 +36,9 @@ def create_app(global_config, **settings):  # noqa: ARG001
     config.include("pyramid_jinja2")
     config.include("pyramid_services")
 
+    # Add our own json renderer that handles datetime objects.
+    config.add_renderer("json_iso_utc", json_iso_utc())
```

Comment:
Perhaps calling it `json_iso_utc` leaks some implementation details which are not completely relevant for the renderer name.

Maybe we could call it `enhanced_json` or something like that, that way we can use it in future to add other customizations and improvements on top of the standard JSON renderer.

Reply:
> Perhaps calling it `json_iso_utc` leaks some implementation details which are not completely relevant for the renderer name.
> 
> Maybe we could call it `enhanced_json` or something like that. That way we can use it in future to add other customizations and improvements on top of the standard JSON renderer.

Yes, good point although for now the plan is to completely replace the current `json` by this and then go back to the `json` name.

If that plan doesn't pan out let's revisit the name then.